lib/main_production.dart

```diff
@@ -15,5 +16,8 @@ Future<void> main() async {
 
   await preAppConfig();
 
-  runApp(App());
+  runApp(App(
+    storedLanguageCode: await getStoredLanguage(),
```

Comment:
We could preload these values in pre_app_config, then either provide them in service_locator, or store them statically in preference_util, and access them directly in app.dart instead of awaiting them in each main app method. 
_This would break the DI principle, but for the app widget that's already broken, so we can do this for convenience and clarity_.